content/concepts/did-ddo.md

```diff
@@ -1,173 +1,602 @@
+## DDO
+
+A DDO in Ocean has these required attributes:
+
+| Attribute         | Type                        | Description                                                                                                    |
+| ----------------- | --------------------------- | -------------------------------------------------------------------------------------------------------------- |
+| **`@context`**    | Array of `string`           | Contexts used for validation.                                                                                  |
+| **`id`**          | `string`                    | Computed as `sha256(address of ERC721 contract + chainId)`.                                                    |
+| **`version`**     | `string`                    | Version information in [SemVer](https://semver.org) notation referring to this DDO spec version, like `4.0.0`. |
+| **`chainId`**     | `number`                    | Stores chainId of the network the DDO was published to.                                                        |
+| **`metadata`**    | [Metadata](#metadata)       | Stores an object describing the asset.                                                                         |
+| **`services`**    | [Services](#services)       | Stores an array of services defining access to the asset.                                                      |
+| **`credentials`** | [Credentials](#credentials) | Describes the credentials needed to access a dataset in addition to the `services` definition.                 |
+
+### Metadata
+
+This object holds information describing the actual asset.
+
+| Attribute                   | Type                                      | Required                          | Description                                                                                                                                                                                       |
+| --------------------------- | ----------------------------------------- | --------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`created`**               | `ISO Date Time string`                    |                                   | Contains the date of the creation of the dataset content in ISO Date Time Format, e.g. `2000-10-31T01:30:00`.                                                                                     |
+| **`updated`**               | `ISO Date Time string`                    |                                   | Contains the date of last update of the dataset content in ISO Date Time Format, e.g. `2000-10-31T01:30:00`.                                                                                      |
```

Comment:
The `created` and `updated` attribute formats listed here don't match what's shown in the "Full Enhanced DDO Example"

Here it's ISO format without the `Z` postfix.

```json
"2000-10-31T01:30:00"
```

But in the example, it's ISO format with a `Z` postfix.  

```json
    "created": "2020-11-15T12:27:48Z",
    "updated": "2021-05-17T21:58:02Z",
```
Which one is correct?  CC @calina-c

Reply:
Both. 
"2000-10-31T01:30:00"  is a datetime object in your local time zone.
"2000-10-31T01:30:00Z" is a datetime object in UTC  (Z = Zero time zone)
"2000-10-31T01:30:00+2" is a datetime object in UTC +2

Reply:
updated this to keep all our date examples consistent within the document, and suggested to always use some timezone designator. Cause without it it's just not precise enough as no dApp knows where the publisher was publishing.